backend/core/entities/docker_entity.py

```
@@ -0,0 +1,330 @@
+    architecture: str = "amd64"
+    os: str = "linux"
+    
+    @property
+    def primary_tag(self) -> Optional[str]:
+        """Get the primary tag (first one)"""
+        return self.tags[0] if self.tags else None
+    
+    @property
+    def repository(self) -> Optional[str]:
+        """Get repository name from primary tag"""
+        if self.primary_tag and ':' in self.primary_tag:
```

Pylint (3.3.7)

[error] Value 'self.primary_tag' doesn't support membership test

(E1135)